nat-lab/tests/utils/multicast.py

```diff
@@ -9,6 +9,8 @@
 def _get_python(connection: Connection) -> str:
     if connection.target_os == TargetOS.Windows:
         return "python"
+    if connection.target_os == TargetOS.Mac:
```

Comment:
For natlab in general we just need to specify `python3`, why do we need to specify the full path for multicast?

Reply:
we are trying to run`bin/multicast.py` from within the test, and the resulting `SshConnection` doesn't seem to find our custom python installation, instead it tries to run the system python installation and fails with `xcode-select: note: no developer tools were found`.
This ensures we are running the correct version of python, [just like when running `list_interfaces_with_router_property.py`](https://github.com/NordSecurity/libtelio/blob/ef49d4aa42e957ed5c40dfb1b99bb3130d8bbb4f/nat-lab/tests/test_vpn.py#L187)